plugins/services/src/js/containers/volume-detail/PodVolumeDetail.js

```diff
@@ -0,0 +1,133 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+import { Link } from "react-router";
+
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection
+  from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import DetailViewHeader from "#SRC/js/components/DetailViewHeader";
+import Page from "#SRC/js/components/Page";
+import ServiceBreadcrumbs from "../../components/ServiceBreadcrumbs";
+import VolumeStatus from "../../constants/VolumeStatus";
+
+class PodVolumeDetail extends React.Component {
+  getSizeLabel() {
+    if (this.props.volume.type === "External") {
```

Comment:
I think we have it in many places. What would you feel about a util?

Reply:
like `isExternal`?

Reply:
We use this pattern in multiple places that is right. How ever instead of adding a util for this I would rather wait for react 16 and Fragments then we could rather do something like:

```JS
import React, { Fragment } from "react";

function SizeLabel({type}) {
    if (type === "External") {
    return <Fragment>Size (GiB)</Fragment>;
  }

  return <Fragment>Size (MiB)</Fragment>;
}

//... And use it like this:

return <SizeLable type="External" />

```

Should I create a Issue for this? 

FYI: [React fragments](https://reactjs.org/docs/fragments.html)